app/models/post.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+end
```

Comment:
Post should have a title and a body, please add them and do not forget to validate

Reply:
fixed